pegasus/sites.v3/code.org/views/school_info.haml

```diff
@@ -0,0 +1,102 @@
+- require 'country_codes'
+- require 'state_abbr'
+- require 'geocoder'
+
+- location = Geocoder.search(request.ip).try(:first)
+-# geocoder sometimes shows localhost's country as RD/Reserved
+- us_ip = location.nil? || ['US', 'RD'].include?(location.country_code.to_s.upcase)
```

Comment:
Late-breaking thought.  I think this might inadvertently show the wrong info to some users.  I don't believe Pegasus does cache variants based on location, and so when it first generates a page for a given user_type/language, it will look up the location and use its US-or-not status, but then all other users hitting the same cache entry will see that result, regardless of their actual location.  To get around this on Pegasus we've made ajax requests to a non-cached location lookup API which we've currently decided is too expensive for the /yourschool page.